ProjectSettings/AudioManager.asset

```diff
@@ -10,8 +10,9 @@ AudioManager:
   Default Speaker Mode: 2
   m_SampleRate: 0
   m_DSPBufferSize: 1024
-  m_VirtualVoiceCount: 512
+  m_VirtualVoiceCount: 1024
   m_RealVoiceCount: 32
+  m_EnableOutputSuspension: 1
```

Comment:
What does this do? Will it stop playing Audio when inaudible? Is this intended?

Reply:
> Enable Output Suspension (editor only)", "When enabled automatically suspends audio output after detecting that the output has been silent for a long duration (editor only).

taken from https://github.com/Unity-Technologies/UnityCsReference/blob/master/Editor/Mono/Inspector/AudioManagerInspector.cs#L29

it was added by Unity when i modified the m_VirtualVoiceCount in the editor

can be removed